SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "SpatialGDKEditorSnapshotGenerator.h"
 
+#include "Algo/Transform.h"
```

Comment:
If we rebase this on top of @tenevdev's change that removes the placeholders, then we no longer need this header.

Reply:
I will do that